### DISCUSSION TITLE
https://mjasion.pl/posts/cloud/how-to-setup-aws-site-to-site-vpn-with-unifi-udm/

### DISCUSSION THREAD

**Discussion by giscus** [mjasion/mjasion#162] [General]
# https://mjasion.pl/posts/cloud/how-to-setup-aws-site-to-site-vpn-with-unifi-udm/

Site-to-Site VPN allows connecting local network with AWS VPC. This blogs is an instruction step by step to establish connection

https://mjasion.pl/posts/cloud/how-to-setup-aws-site-to-site-vpn-with-unifi-udm/

**Comment by brajshoriegzon** [comment]
Hi, 

thanks for the article. I'm trying to follow your implementation but there's a few issues I'm running into.

When I enable route propagation on my route tables, the propagation doesn't happen, I have to add the routes manually, do you have any idea why that would happen?

I also tested the implementation using both virtual private gateways and transit gateways but I can't get my network to communicate with my AWS VPC(s). The tunnels remain down all the time even though I am pretty certain the configuration implemented should be correct. 

On the Unifi console under Remote IP, should that be the Outside IP Address of one of the tunnels on AWS?

I suspect some kind of configuration on Unifi on our end is blocking the communication between the sites, but I'm not quite sure why, I'm not the one who have set that up

**Reply by mjasion** [reply]
Hey @brajshoriegzon 

Sorry for late response, I didn't manage to find a time to reproduce your issue :( Answearing your question about route propagations, AFAIR they are propagated when the IPSec tunnel is established. Until your gateway not make a connection, then route will not be propagated.

About Remote IP in Unifi and Outside IP in AWS - you are right. Please check your firewall rules on EC2 instance and on Unifi, by temporary disabling them. It is a common issue that firewall rules on router blocks egress to AWS.
